aci/data_source_aci_fvsubnet.go

```diff
@@ -38,6 +38,12 @@ func dataSourceAciSubnet() *schema.Resource {
 				},
 			},
 
+			"ip_data_plane_learning": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
```

Comment:
Can you change the attributes that are not required to retrieve the information to be computed only? 

```suggestion
				Computed: true,
```

Reply:
Could you provide this for the other optional attributes also? This is a change we have been consistently been making for updating modules, because provided configuration for optional attributes in data-sources will always be overwritten by the information retrieved from apic.

Reply:
Sure, I'll fix that.